Sources/PrimerSDK/Classes/Core/Payment Services/CreateResumePaymentService.swift

```diff
@@ -97,4 +98,34 @@ internal class CreateResumePaymentService: CreateResumePaymentServiceProtocol {
             }
         }
     }
+
+    /**
+     * Completes a payment using the provided JWT token and URL.
+     *
+     * This private method performs an API call to complete a payment, using a decoded JWT token for authentication
+     * and a URL indicating where the completion request should be sent.
+     *
+     * - Parameters:
+     *   - clientToken: A `DecodedJWTToken` representing the client's authentication token.
+     *   - completeUrl: An `URL` indicating the endpoint for completing the ACH payment.
+     *
+     * - Returns: A `Promise<Void>` that resolves if the payment is completed successfully, or rejects if there is
+     *            an error during the API call.
+     */
+    func completePayment(clientToken: DecodedJWTToken, completeUrl: URL) -> Promise<Void> {
```

Comment:
[q]: wouldn't it make more sense to send some params to be mapped to request body or maybe request body itself? This seems to be a generic function that creates body for only one use case.